iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredDepartureDetails.swift

```diff
@@ -149,7 +149,7 @@ struct StopDetailsFilteredDepartureDetails: View {
         ForEach(Array(statuses.enumerated()), id: \.offset) { index, row in
             VStack(spacing: 0) {
                 OptionalNavigationLink(
-                    value: row.navigationTarget,
+                    value: nil,
```

Comment:
An `OptionalNavigationLink(value: nil)` is just its label, so if this never needs to be a link anymore then it should probably not be an `OptionalNavigationLink`.

Reply:
The UI to replace the status rows isn't implemented yet, this is just a placeholder so I wasn't paying much attention to it, just made it a little prettier and got rid of the nav link.